rules/repositories.bzl

```diff
@@ -27,15 +27,15 @@ def rules_ios_dependencies():
     _maybe(
         git_repository,
         name = "build_bazel_rules_apple",
```

Comment:
I think `build_bazel_apple_support` might also need a bump

Reply:
nope we are still pointing at HEAD of master